src/test/java/com/vackosar/gitflowincrementalbuild/BaseRepoTest.java

```diff
@@ -26,13 +32,40 @@
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
     public BaseRepoTest() {
+        this(false);
+    }
+
+    public BaseRepoTest(boolean useSymLinkedFolder) {
         this.normalOut = System.out;
+        this.useSymLinkedFolder = useSymLinkedFolder;
     }
 
     @Before
     public void before() throws Exception {
         init();
-        localRepoMock = new LocalRepoMock(temporaryFolder.getRoot(), false);
+        File repoBaseFolder = temporaryFolder.getRoot();
+
+        // place repo in a sym-linked folder if requested by the concrete test class
+        if (useSymLinkedFolder) {
+            Path linkTarget = repoBaseFolder.toPath().resolve("link-target");
+            Path link = repoBaseFolder.toPath().resolve("link");
+            Files.createDirectory(linkTarget);
+
+            // - creation of most links requires elevated rights on windows, including the ones
```

Comment:
Interesting

Reply:
And a bit annoying. On Win10 _Home_ you can't even grant yourself the permission as a admin.
See also: https://stackoverflow.com/questions/23217460/how-to-create-soft-symbolic-link-using-java-nio-files